#Robot Parameters
Robot name: Dilyn
wheel_diameter: 0.08
wheel_base: 0.175
leftMaxRPM: 107
rightMaxRPM: 107
leftTicksPerRotation: 990
rightTicksPerRotation: 990
leftForwardPin: 16
leftBackwardPin: 17
rightForwardPin: 18
rightBackwardPin: 19
